src/RepositoryAssertions.php

```diff
@@ -18,6 +18,8 @@
  */
 final class RepositoryAssertions
 {
+    private static string $countWithoutCriteriaDeprecationMessagePattern = 'Passing the message to %s() as second parameter is deprecated. Use third parameter.';
```

Comment:
in all  PHPUnit assertions, "message" parameter is the last parameter, so I wanted to keep the same logic.

Reply:
👍🏻